src/corehost/cli/hostpolicy.cpp

```diff
@@ -100,8 +100,28 @@ int run(const arguments_t& args)
     pal::pal_clrstring(probe_paths.native, &native_dirs_cstr);
     pal::pal_clrstring(probe_paths.resources, &resources_dirs_cstr);
 
-    pal::pal_clrstring(resolver.get_fx_deps_file(), &fx_deps);
-    pal::pal_clrstring(resolver.get_deps_file() + _X(";") + resolver.get_fx_deps_file(), &deps);
+    if (resolver.get_fx_definitions().size() >= 2)
+    {
+        // Use the first fx to define FX_DEPS_FILE
```

Comment:
I'm not sure this is right.  I think it might make more sense to be the `root` framework.  There is code that expects this to be `Microsoft.NETCore.App`.  See 

[https://github.com/dotnet/cli/blob/cc80822c16b955a1f664f8a65ed2c76c3c52a824/src/Microsoft.DotNet.Cli.Utils/Muxer.cs#L17-L24](https://github.com/dotnet/cli/blob/cc80822c16b955a1f664f8a65ed2c76c3c52a824/src/Microsoft.DotNet.Cli.Utils/Muxer.cs#L17-L24)

or any of these places:

[https://github.com/search?q=org%3Adotnet+FX_DEPS_FILE&type=Code](https://github.com/search?q=org%253Adotnet+FX_DEPS_FILE&type=Code)

Reply:
Thanks. Changed to root